src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -478,6 +478,7 @@ public String getDefaultBranch() throws IOException, InterruptedException {
         String url = UriTemplate.fromTemplate(REPO_URL_TEMPLATE + nodePath)
```

Comment:
I think you need to add the parameter to the template like so:
> String url = UriTemplate.fromTemplate(REPO_URL_TEMPLATE + nodePath + "{?pagelen}")

I tested locally with logging the URL

Reply:
Good catch 😅

Reply:
@dblanchette protip you can use suggestions:

```suggestion
        String url = UriTemplate.fromTemplate(REPO_URL_TEMPLATE + nodePath + "{?pagelen}")
```

````
```suggestion
        String url = UriTemplate.fromTemplate(REPO_URL_TEMPLATE + nodePath + "{?pagelen}")
```
````

Reply:
Thanks for the tip!